tls/s2n_post_handshake.c

```diff
@@ -32,26 +32,28 @@ int s2n_post_handshake_recv(struct s2n_connection *conn)
     uint32_t message_length;
     S2N_ERROR_IF(conn->actual_protocol_version != S2N_TLS13, S2N_ERR_BAD_MESSAGE);
 
-    GUARD(s2n_stuffer_read_uint8(&conn->in, &post_handshake_id));
-    GUARD(s2n_stuffer_read_uint24(&conn->in, &message_length));
+    while(s2n_stuffer_data_available(&conn->in)) {
+        GUARD(s2n_stuffer_read_uint8(&conn->in, &post_handshake_id));
+        GUARD(s2n_stuffer_read_uint24(&conn->in, &message_length));
```

Comment:
Prior to this loop, we should add a check for sufficient stuffer data available `s2n_stuffer_data_available(&conn->in) > 0.` 

Also nit: space after while, `while (s2n_stuffer_data_available(&conn->in))`.